docs/modules/3d-tiles/api-reference/tiles-3d-loader.md

```diff
@@ -59,13 +59,24 @@ const tileset3d = new Tileset3D(tilesetJson, {
   onTileLoad: tile => console.log(tile)
 });
 
-const viewport = new WebMercatorViewport({latitude, longitude, zoom, ...});
-tileset3d.update(viewport);
+const viewport = new WebMercatorViewport({
+  width: 600,
+  height: 400,
+  latitude: 40.7067584, 
+  longitude: -74.0115413, 
+  zoom: 17
+});
+tileset3d.selectTiles(viewport);
 
 // visible tiles
 const visibleTiles = tileset3d.tiles.filter(tile => tile.selected);
 // Note that visibleTiles will likely not immediately include all tiles
 // tiles will keep loading and file `onTileLoad` callbacks
+
+// To fully load a tileset, repeatedly select tiles until the tileset is loaded
+while (!tileset3d.isLoaded()) {
```

Comment:
just a thought for the future, 

maybe we should have a `loadAllTiles(viewport)` convenience function?
normal pattern is that application asynchronously responds to onTileLoad callbacks.
That is good for UI applications, but less so for an analytics type (code only) app.